web/docs/writingguide.md

```diff
@@ -0,0 +1,158 @@
+- **When you ask people for feedback, tell reviewers:**
+  - **What you're trying to do.**
+  - **What your fears are.**
+  - **Which balances you're trying to strike.**
+- **Do your best to come up with a good and simple way to say something.** Again, this will help the reviewer focus on high-level issues instead of rephrasing your sentences.
+- **Read and correct your text several times before submitting it (preferably with some time between the readings).** This is similar to proof-reading, but has more to do with content and the communication style than with grammar.
+  A time offset is beneficial because it removes the text from your short-term memory, helping you view it more objectively.
+- **It's OK to ask AI to improve your text.** Just make sure to check it and correct. You should always sign off on the last version.
+- **When someone reports a problem, there is almost always a problem**, even if the solution they proposed isn't quite right. Keep asking follow-up questions to learn more.
+- People need to feel safe asking questions when contributing/reviewing content. Here's how you can do that:
+  - **Thank people for their contributions/reviews, even if you're feeling grumpy.** For example:
+    - "Great question!"
+    - "Thanks for taking the time to explain. 🙂"
+    - "This is actually intentional, but thanks for taking the time to contribute. 😊"
+  - **Listen to what people are saying and mirror if you're not sure you're understanding correctly.** This can help validate people's feelings and experiences, while also understanding if _you're_ understanding _them_ correctly.
+  - **Use a lot of positive and empathetic emojis.** It's always better to seem a little strange than mean or impatient.
+    This is primarily applicable for Wasp team members speaking to outside contributors. Since most of the core team knows each other pretty well, there's no need to go overboard with the emojis and pleasentries.
+  - **Kindly communicate rules/boundaries.** If someone behaves in a way that's abusive/inappropriate, respond only with kindness and maturity, but also make it clear that this behavior is not acceptable and what will happen (according to the code of conduct) if they continue behaving poorly.
+- **All docs must go through the review cycle, preferably with more than a single reviewer.** Different people focus on different things. Some of us are great at coming up with examples, others easily come up with analogies and explain complex topics, some have a clear and concise writing style, etc. Therefore, try to get at least two or three people to review your document.
+
+## Possible improvements
+
+- Some parts of our docs don't follow all the guidelines outlined in this document. There's no need to start fixing all the issues right away, we can slowly improve the docs as we edit them.
+- We've talked about having a git repo with all the example code so it's easier to copy-paste, test, and maintain.
```

Comment:
Is this about example apps? Those are kind of separated from docs though, right? Or is is this about something else?